static/app/views/settings/organizationIntegrations/integrationCodeMappings.spec.jsx

```diff
@@ -170,4 +175,29 @@ describe('IntegrationCodeMappings', function () {
       })
     );
   });
+
+  it('switches default branch to the repo defaultBranch', async () => {
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/integrations/${integration.id}/repos/`,
+      body: {
+        repos: [
+          {
+            id: repos[0].id,
+            identifier: repos[1].name,
+            defaultBranch: 'main',
+          },
+        ],
```

Comment:
Should we add the case when there's multiple repos matching?

Reply:
we shouldn't ever have any since all the git providers that matter use org/unique-name

Reply:
So we're not doing string matching. Got it.